janitor/functions/min_max_scale.py

```diff
@@ -8,11 +10,12 @@
 @deprecated_alias(col_name="column_name")
 def min_max_scale(
     df: pd.DataFrame,
-    old_min=None,
```

Comment:
I thought there should be a _typing.py script that could contain some common typing.